tasks/build-css.js

```diff
@@ -11,22 +11,117 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+import path from 'path';
+import fs from 'fs';
+import { fileURLToPath } from 'url';
+import { createRequire } from 'node:module';
+
 import fg from 'fast-glob';
-import { processCSS } from './css-tools.js';
```

Comment:
We didn't really need this abstraction since build-css is the only script calling css-tools. I pulled that functionality directly into this script as a single-source-of-truth.